File: example/private-sources/encrypt-classified.js
const crypto = require('crypto')

const algorithm = 'aes-256-cfb8'
const KEY_SIZE = 32
const BLOCK_SIZE = 16

const key = crypto.randomBytes(KEY_SIZE)
const iv = crypto.randomBytes(BLOCK_SIZE)

const plaintext = Buffer.from('If someone else can run '
  + 'arbitrary code on your computer, it\'s not YOUR '
  + 'computer any more. - Rich Kulawiec')

const cipher = crypto.createCipheriv(algorithm, key, iv)

const ciphertext = cipher.update(plaintext)

module.exports = { key, iv, ciphertext }

File: example/private-sources/encrypt-classified.py
# this file is now unused
from Cryptodome.Cipher import AES
from Cryptodome import Random

key = Random.new().read(AES.key_size[2])
iv = Random.new().read(AES.block_size)

plaintext = (b"If someone else can run arbitrary"
  b" code on your computer, it\'s not YOUR computer any more. -"
  b" Rich Kulawiec")

encrypter = AES.new(key, AES.MODE_CFB, iv)
print(key.hex(), iv.hex(), encrypter.encrypt(plaintext).hex())
